reporter/src/main/resources/templates/notice/default.ftl

```diff
@@ -62,10 +62,12 @@ ${exceptionText}
         [/#if]
     [/#if]
 [/#list]
-[#-- Add the licenses of all dependencies. --]
+[#-- Add the licenses of all packages. --]
 [#if packages?has_content]
+[#if hasNoticeProjectLicenses]
```

Comment:
I think the indentation is wrong for this element?!

Reply:
Hah, good eyes, fixed!